src/components/organisms/cards/cards.scss

```
@@ -0,0 +1,19 @@
+@use "../../variables" as *;
+@use "../../molecules/card/card.scss" as *;
+
+.cards-container {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 100px;
+  @media (max-width: $breakpoint-md) {
+    display: block;
+    margin-bottom: 40px;
+  }
+  .card {
+    width: 31%;
+    @media (max-width: $breakpoint-md) {
+      width: 100%;
+      margin-bottom: 40px;
+    }
+  }
+}
```

```suggestion
.cards-container {
  display: flex;
  justify-content: space-between;
  margin-bottom: 100px;
  gap: 24px;
  @media (max-width: $breakpoint-md) {
    display: block;
    margin-bottom: 40px;
  }
  .card {
    flex: 1;
    @media (max-width: $breakpoint-md) {
      width: 100%;
      margin-bottom: 40px;
    }
  }
}
```

こうするとよりスマートにかけます。
flexプロパティは、flexコンテナの中でどの程度伸縮するかを決めるもので、gapはflexアイテム間の余白を調節します。
新教材の[ここ](https://shinonome-inc.notion.site/Flexbox-1-e28e57a82d004507bfef1179b8fd6845)に詳しく書いてあるので確認してみてください！